plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -1157,6 +1158,24 @@ else if (avroSchemaLiteral != null) {
         tableProperties.put("numFiles", "-1");
         tableProperties.put("totalSize", "-1");
 
+        // Extra properties
+        getExtraProperties(tableMetadata.getProperties())
```

Comment:
`tableMetadata.getProperties()` is called over `10` times in `getEmptyTableProperties()` method.
Can you pls extract it to a variable in a preparatory commit?

Reply:
But won't it reduce the code readability ? Like we might need to introduce a new variable with `tableProperties` - which might conflict with the existing variable ?

Reply:
I don't think that adding a new variable `tableMetadataProperties` will reduce the readability of the code.
You can however disregard this request because it is rather a nit.